packages/dds/sequence/src/sequence.ts

```diff
@@ -461,7 +461,10 @@ export abstract class SharedSegmentSequence<T extends ISegment>
     protected getGCDataCore(): IGarbageCollectionData {
         // Create a SummarySerializer and use it to serialize all the cells. It keeps track of all IFluidHandles that it
         // serializes.
-        const serializer = new SummarySerializer(this.runtime.channelsRoutingContext);
+        const serializer = new SummarySerializer(
```

Comment:
Maybe add a helper method on SharedObject to create such serializer, if every DDS needs to do so?

Reply:
We could create the serializer in SharedObject and pass it to getGCDataCore.

Reply:
Let me do that in a separate PR.